From abfdfa8ec247d31d40dcbbf57c88ade3a276cb0d Mon Sep 17 00:00:00 2001
From: Devi Priya <quic_devipriy@quicinc.com>
Date: Fri, 17 Feb 2023 19:11:03 +0530
Subject: [PATCH 166/281] dt-bindings: clock: qcom,a53pll: add IPQ9574
 compatible

IPQ9574 uses A73 PLL of type Huayra.
Add the IPQ9574 A73 compatible to A53 bindings as the PLL
properties match with that of A53

Change-Id: Ia2bf1352029676424292a75e23a969e532f23664
Signed-off-by: Devi Priya <quic_devipriy@quicinc.com>
Acked-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
Signed-off-by: Manikanta Mylavarapu <quic_mmanikan@quicinc.com>
---
 Documentation/devicetree/bindings/clock/qcom,a53pll.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/clock/qcom,a53pll.yaml b/Documentation/devicetree/bindings/clock/qcom,a53pll.yaml
index 5df07a1f4884..fa74c43417a8 100644
--- a/Documentation/devicetree/bindings/clock/qcom,a53pll.yaml
+++ b/Documentation/devicetree/bindings/clock/qcom,a53pll.yaml
@@ -19,6 +19,7 @@ properties:
       - qcom,ipq5332-a53pll
       - qcom,ipq6018-a53pll
       - qcom,ipq8074-a53pll
+      - qcom,ipq9574-a73pll
       - qcom,msm8916-a53pll
       - qcom,msm8939-a53pll
 
-- 
2.17.1

